scaffolds/ruby/docker.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ENABLE_TTY=${ENABLE_TTY:--t}
+IMAGE="code-retreat-ruby-env"
+VERSION="1"
+TAG="$IMAGE:$VERSION"
+
+docker images | grep -q "$IMAGE[ ]*$VERSION" || docker build -t $TAG .
```

Comment:
Improvement suggestion: stop if building the image fails:

```suggestion
docker images | grep -q "$IMAGE[ ]*$VERSION" || docker build -t $TAG .  || { echo "Failed to build image"; exit 1; }
```